### PR TITLE
fix issues and make dynamic

### PR DESCRIPTION
**Pull request by astikdev**
fix issues and make component dynamic.

### CODE REVIEW

File: src/Organisms/GradientLogoFrame/GradientLogoFrame.js [hunk 1]
```diff
@@ -4,17 +4,15 @@ import GradientShadowFrame from '../../Molecules/GradientShadowFrame/GradientSha
...
+  const {className, src, alt, shadow, gradient} = props
   return (
-    <BackgroundFrame {...props}> // why same propos passing 3 component ?
-      <GradientShadowFrame {...props}  /> // why same propos passing 3 component ?
-      {props?.logoImage && (
-        <Img {...props}/> // why same propos passing 3 component ? propos with spred oprater only be used in atoms
-      )}
+    <BackgroundFrame>
+      <GradientShadowFrame shadow={shadow} gradient={gradient} />
+      {props?.src && <Img className={className} src={src} alt={alt} />}
```
Comment (Deeptrentium) [inline src/Organisms/GradientLogoFrame/GradientLogoFrame.js]: add spared props here

File: src/Templates/SignUp/SignUp.js [hunk 2]
```diff
@@ -9,29 +9,21 @@ const SignUp = () => {
   return (
     <>
       <GradientLogoFrame
```
Comment (Deeptrentium) [inline src/Templates/SignUp/SignUp.js]: if its a gradientLogofrem that why we should pass gradient as true change the propose or else change the file name

File: src/atoms/Button/Button.js [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
-        {...props}
-        >
-              {props?.ImageSRC && <img src={props.ImageSRC} alt={props.ImageALT} />} // props name first name naver capital
-              <span className='button_text'>{props.text}</span> // how I can change css of this text 
-        </button>      
-
-    </>
+    <button className={props.className || 'form_submit_Button'} {...props}>
+      {props?.imageSrc && <img src={props.imageSrc} alt={props.imageAlt} />}
+      {props?.textClassName ? (
```
Comment (Deeptrentium) [inline src/atoms/Button/Button.js]: remove ternary operator

File: src/atoms/Img/Img.js [hunk 4]
```diff
@@ -4,13 +4,14 @@ import './Img.css'
 const Image = (props) => {
   return (
     <>
```
Comment (Deeptrentium) [inline src/atoms/Img/Img.js:6]: remove unused tag

File: src/atoms/Input/Input.js [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
   return (
     <div className='main'>
       <div className='input_type'>
-        {props?.imageSRC && <img src={props.imageSRC} alt={props.ImageALT} />}
+        {props?.imageSrc && <img src={props.imageSrc} alt={props.imageAlt} />}
         <Field
           name={props.name}
           type={props.type}
-          placeholder={props?.InputPlaceholder} // add extra propos first latter navew capital in propos
+          placeholder={props?.inputPlaceholder}
```
Comment (Deeptrentium) [inline src/atoms/Input/Input.js:13]: add spared operator props here

File: src/atoms/Link/Link.js [hunk 6]
```diff
@@ -3,9 +3,7 @@ import './Link.css'
 
 const Link = (props) => {
   return (
-    <span> // what is the use of span tag
       <a className={props.className} href={props?.href || '#'}>{props.text}</a>
```
Comment (Deeptrentium) [inline src/atoms/Link/Link.js]: classname is not mandatory

File: src/atoms/Text/Text.js [hunk 7]
```diff
@@ -3,7 +3,7 @@ import './Text.css'
 
 const Text = (props) => {
   return (
-    <span className={props.className} >
+    <span className={props?.className} >
```
Comment (Deeptrentium) [inline src/atoms/Text/Text.js]: use li props?.className || ' '

**Review by Deeptrentium — CHANGES_REQUESTED** [review]

File: src/Organisms/GradientLogoFrame/GradientLogoFrame.js (hunk 1 above)
Comment (astikdev) [reply src/Organisms/GradientLogoFrame/GradientLogoFrame.js]: Show Error
**img is a void element tag and must neither have `children` nor use `dangerouslySetInnerHTML`.**

**Pull request merged**